src/data/content-items/data-source.js

```diff
@@ -440,7 +507,8 @@ export default class ContentItem extends oldContentItem.dataSource {
               return {
                 __typename: 'TextNote',
                 id: createGlobalId(id, 'TextNote'),
-                allowsCustomNote: custom === 'True',
+                allowsComment: custom === 'True',
```

Comment:
Why "comment"? Might be nit-picky, but I'm not sure that changing this makes sense. The UI in Rock says "Add Note Field" and the UI in the app says "Add a Note". If everywhere else it says "note" and we change it to "comment" in the code, that seems confusing.

Reply:
Yea...I changed it because to me the "Note" language here is confusing. It's starting to get messy with Notes in Rock as well which are a Rock data type and another completely separate idea. So currently here's the glossary:

**Rock:**
- Sermon Notes - matrix of data attached to a content item
- "Add Note Field" - boolean attached to each matrix line item
- Note - rock data type, what we use to store the custom saved text from users

**API**
- `SermonNote` - block in sermon notes corresponding to a type of text
- `TextNote` - `SermonNote` that shows fill in the blanks
- `ScriptureNote` - `SermonNote` that shows bible verses
- comment - custom saved text attached to a particular `SermonNote`

**App**
- "Add a Note" - UX label that allows users to add custom text per sermon note block, and save it.

Do you still think the field should be called "custom note"? If not, you're like me, but what should it be besides comment? Or should we take this opportunity before anything is enabled to change the name of something else in the stack?

Reply:
I'm not necessarily sold that it has to be "custom note", but I do kind of think it should be some form of "note". Like you said, we're even saving this text as a Note in Rock using a NoteType specifically created for this purpose. To me it seems like moving away from something related to "note" would only end up being more confusing in the long run, especially for anyone else that might come across this code later.

We've got `SermonNote` and `TextNote` and `ScriptureNote` ... so `CustomNote` or something similar seems to make sense.

Reply:
I don't think it should be XxxNote because it's not the same as `SermonNote`, `TextNote`, or `ScriptureNote`. 

Also, the API should not be concerned at all with what Rock calls things. It should, however, be consistent with its own schema. It should be agnostic to rock data types.

So if you think we should use `customNote` for that field, we should rename the `SermonNote` interface and the two derived types.

Reply:
I don't necessarily disagree that we shouldn't care what Rock calls things but I do think we need to be as clear as possible about the type of thing we're saving and I believe where we're saving it to does make a difference. You wouldn't call something a "note" or a "person" that we're saving to the `Interactions` table. 

So the "comment" as it is now is actually a property of the `TextNote` and `ScriptureNote`, right? It's not technically a type of "note" as the schema defines "note". 

Tell me, what would you recommend we rename the `SermonNote`, `TextNote`, and `ScriptureNote` to if we were going to go that route?